1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -490,3 +495,33 @@ const quotes = [
     author: "Zig Ziglar",
   },
 ];
+
+
+/*
+1. Render the first quote from the array
+2. Learn how to pick a random quote from the array using Math.random
+3. Render a random quote from the array when the screen first loads
+4. When you click the "next" button on the screen it should change the quote on the screen
+*/
+
+const quotesHolder = document.querySelector(".quotes-holder")
+const quoteText = document.querySelector("#quote");
+const quoteAuthor = document.querySelector("#author");
+
+function getRandomQuotes(quoteList) {
+  const randomNumber = Math.floor(Math.random() * quotes.length);
```

Comment:
The exercise already came with a random quote picker code, so there's a bit of repetition here (nothing wrong with implementation though!)